src/css/components/_live.scss

```diff
@@ -12,3 +11,55 @@
   width: auto;
   text-align: left;
 }
+
+// hide the LiveDisplay when not live or when
+// the new liveui is in use
+.video-js:not(.vjs-live) .vjs-live-control,
+.video-js.vjs-liveui .vjs-live-control {
+  display: none;
+}
+
+// css for the new live ui below
+.video-js .vjs-seek-to-live-control {
+  cursor: pointer;
+  @include flex(none);
+  display: inline-flex;
+  height: 100%;
+  margin-left: 0.7em;
+  font-size: 1em;
+  line-height: 3em;
+  width: auto;
```

Comment:
we should add a `min-width: 4em;` here, so, it'll be at least as big as all other components.